### DISCUSSION TITLE
2-Way Camera Audio

### DISCUSSION THREAD

**Discussion by benregan** [homebridge-simplisafe3/homebridge-simplisafe3#234] [General]
Up front, thank you Devs! This plugin is awesome! 

I am curious, it seems the devs working on the Ring homebridge plugin were able to get 2-way audio working through HomeKit... I'm wondering, is there a limitation in the way that Simplisafe produces the streams as well as the audio, in that it is done differently than Ring? Or is it feasible to get it working for this plugin as well... given someone to take up the "challenge" :D 

 Super solid plugin by the way! Stoked to _Finally_ have some support in HomeKit... even if it is "unofficial" :)

**Comment by shamoon** [comment]
Hi @benregan thanks!

Please see #230 , but TLDR unfortunately not likely any time soon (no known way to do this via API)

**Reply by benregan** [reply]
That's about what I thought. Heh. Thank you for the hard work on this plugin, and the quick reply. You guys are awesome!
